builtin/client/finish_core_api.lua

```diff
@@ -0,0 +1,10 @@
+
+-- old non-method sound functions
+
+function core.sound_stop(handle, ...)
+	return handle:stop(...)
+end
+
+function core.sound_fade(handle, ...)
+	return handle:fade(...)
+end
```

Comment:
why isn't this in misc.lua?

Reply:
Because misc.lua is loaded after all the other lua files.

But considering the sort of functions defined in misc.lua, this makes sense. Moved.